roles/ceph-common/templates/ceph.conf.j2

```diff
@@ -135,7 +135,11 @@ keyring = /var/lib/ceph/radosgw/{{ cluster }}-rgw.{{ hostvars[host]['ansible_hos
 rgw socket path = /tmp/radosgw-{{ hostvars[host]['ansible_hostname'] }}.sock
 log file = /var/log/ceph/{{ cluster }}-rgw-{{ hostvars[host]['ansible_hostname'] }}.log
 rgw data = /var/lib/ceph/radosgw/{{ cluster }}-rgw.{{ hostvars[host]['ansible_hostname'] }}
-rgw frontends = civetweb {{ radosgw_civetweb_options }}
+{% if ip_version == 'ipv6'  %}
+rgw frontends = civetweb port=[{{ hostvars[host]['ansible_'+ monitor_interface]['ipv6'][0]['address'] }}]:{{ radosgw_civetweb_port }} {{ radosgw_civetweb_options }}
```

Comment:
@SirishaGuduru please, take a look at [this](https://github.com/SirishaGuduru/ceph-ansible/blob/341af20e6820a924df11cb983aaa674a07954606/roles/ceph-common/templates/ceph.conf.j2#L67-L73) to make it the same way